api_server.py

```
@@ -228,37 +241,50 @@ def health():
 def root():
     try:
         now = datetime.now()
```

Ruff (0.15.0)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)